routes/api.js

```diff
@@ -78,6 +80,7 @@ api.delete('/user/:secret', function (req, res) {
     })
 })
 
+// Route for removing friendship
 api.delete('/friendships/user/:secret', async function (req, res) {
```

Comment:
This route seems kind of weird, since it basically implies that `user` is a sub-resource of friendships. I feel like it should be the other way around -- friendships are a sub-resource of users.

that way, the route would be DELETE `/user/:secret/friendships/:friendId`

This would also force POST `/friendships/` to turn into POST `/user/:secret/friendships` which makes sense honestly. This also gives us the secret, with which to authenticate the user before executing the friendship creation